nvflare/fuel/utils/pipe/cell_pipe.py

```diff
@@ -178,18 +179,22 @@ def __init__(
         super().__init__(mode)
         self.logger = logging.getLogger(self.__class__.__name__)
 
+        self.site_name = site_name
+        self.token = token
+        self.root_url = root_url
+        self.secure_mode = secure_mode
+        self.workspace_dir = workspace_dir
+
+        sysvarname_placeholders = ["{" + varname + "}" for varname in dir(SystemVarName)]
+        if any([arg in sysvarname_placeholders for arg in [site_name, token, root_url, secure_mode, workspace_dir]]):
+            return
```

Comment:
why do we need this part?

Reply:
This file doesn't seem relevant to this PR. If you see any changes needed, please open a different PR.

Reply:
removed